### DISCUSSION TITLE
Kicad and/or Gerbers for NanoBCU

### DISCUSSION THREAD

**Discussion by cortices** [OpenKNX/OpenKNX#9] [General]
hello,

I'm outside of Europe, so it's not currently possible for me to buy NanoBCUs and other parts from you.
I also think it would be easier for me to just get some PCBs made and assemble them myself.

I notice that the NanoBCU schematic is in this repo, but not board files.
Would it be possible to upload a Kicad ECAD file and/or gerbers ZIP so I can make some NanoBCUs myself?

**Comment by Ing-Dom** [comment]
no, sorry. which country ?

**Comment by cortices** [comment]
Australia.

**Comment by Ing-Dom** [comment]
please contact me by mail you find in the shop.
